packages/backend/src/api/endpoints/InvitationApi.ts

```diff
@@ -10,11 +10,16 @@ type CreateParams = {
   publicMetadata?: UserPublicMetadata;
 };
 
+type FilterParams = {
```

Comment:
🔧 This should probably be `GetInvitationListParams` in case we want to add more options.